feature/selector.py

```diff
@@ -597,25 +681,25 @@ def calculate_statistics(scores: pd.DataFrame,
     scores_df = scores[columns].copy()
     selected_df = selected[columns].copy()
 
+    # Group by feature for CV results
+    scores_df = scores_df.groupby(scores_df.index).mean()
```

Comment:
Minor thought: we might also want to quantify the variability of the scores across folds, so something like stddev might also be useful here in addition to mean. However, it would make the output different compared to a non-CV benchmark, so I'm not entirely sure if it's worth it.

Reply:
Yes, also thought about that. I don't think it's obvious how it should be used so decided to rather keep the output format unchanged. Note that the user can still calculate more complicated statistics manually based on the results returned by benchmark().